content/docs/machine-learning/quickstart.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 2
+title: Get Started with Kubeflow
+description: Learn how to get started with Civo's Kubeflow via an example.
+---
+
+<head>
+  <title>Kubeflow quickstart | Civo Documentation</title>
+</head>
+
+In this quickstart mainly focused towards Pipelines and Notebooks, we take a look at a simple example to train and run a model using Kubeflow Pipelines and Notebooks.
+
+## Before you start
+
+Make sure you can log into your Kubeflow cluster by following the instructions at [Logging into your cluster](kubeflow-dashboard.md#logging-into-your-cluster).
+
+Make sure you can create a notebook and log into the notebook by following the instructions at [Creating a Kubeflow Notebook](creating-a-new-kubeflow-notebook.md).
+
+## Run a pipeline
+
+We will first compile a pipeline DSL that will train a model using the MNIST dataset. Notice that this pipeline mainly has the following coponents which are run: hyperparameter tuning with Katib, creating a new volume for training, run a training job and finally serve the model using KServe.
```

Comment:
Link to the dataset? Define MNIST?

Reply:
got it

Reply:
There is no official website afaik, the original paper could potentially be cited